help_to_heat/frontdoor/views.py

```diff
@@ -637,14 +638,13 @@ def save_post_data(self, data, session_id, page_name):
 
         uprn = epc_details.get("uprn")
         address = epc_details.get("address")
-        heat_source = epc_details.get("mainheat-description")
 
         epc_data = {
             lmk_field: lmk,
             address_field: address,
             epc_details_field: epc_details,
+            recommendations_field: recommendations,
             uprn_field: uprn if uprn is not None else "",
-            property_main_heat_source_field: heat_source if heat_source is not None else "",
```

Comment:
chose to deprecate this as it's now non-standard and may be confusing why this field only is exported like this

Reply:
Is this something we should note on the ticket/check with DESNZ?